tests/src/TestShapeMocks.cpp

```diff
@@ -49,7 +49,7 @@ TEST_CASE("MockShape", "[mocks::MockShape]")
   GIVEN("A mock shape with function objects returning values")
   {
     mocks::MockShape shape;
-    size_t area_f_count = 0;
+    int area_f_count = 0;
```

Comment:
is this the classic "integer literal and size_t have different sizes on 64-bit Windows" issue? If so, you should be able to use `REQUIRE(area_f_count == static_cast<size_t>(1))`.

(But as I have no access to Windows, I can't tell whether this is the issue, or whether that resolves it. Hence asking...)